src/i18n/es.js

```diff
@@ -27,6 +27,19 @@ const urlSiteContact = `${urlSite}/contacto/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 
 export default {
+  campaign_history: {
+    status: {
+      active: 'Activo',
+      inactive: 'Activo no asociado a listas',
+      pending: 'Pendiente',
+      stand_by: 'Cumplir',
```

Comment:
How about `Pendiente`?

Reply:
Here in doppler is like this 
![image](https://user-images.githubusercontent.com/2439363/72150645-c14d7500-3384-11ea-86ea-88d630b38391.png)
`En espera de ampliacion...`

Reply:
From doppler Master subscriber we have in spanish, we should respect the case:
Activo
Activo no Asociado a Listas
En Espera de Ampliación...
Removido por el Cliente
Removido por el Suscriptor
Removido por No Aperturas
Removido por Rebote Hard
Removido por Rebote Soft

Reply:
In English the states with uppercased letters are as follows:
Active
Active not Associated to List
On Hold waiting for Upgrade
Removed by Client
Removed by Hard-Bounced
Removed by Never-Opened
Removed by Soft-Bounced
Removed by Subscriber

Reply:
fixed